src/absil/il.fs

```diff
@@ -8,14 +8,18 @@ module (*internal*) Microsoft.FSharp.Compiler.AbstractIL.IL
 
 
 open Internal.Utilities
+#if FABLE_COMPILER
```

Comment:
Please use ``FSLANG_NO_IMPLICITOPEN`` to indicate a dialect of F# that doesn't have implicit opening of these namespaces, thanks